docker/pdns/run.sh

```diff
@@ -39,8 +39,7 @@ load-modules=liblmdbbackend.so
 launch=lmdb
 lmdb-filename=/lmdb/instance-$instance/db
 lmdb-shards=1
-lmdb-random-ids=yes
-lmdb-flag-deleted=yes
+lmdb-lightning-stream
```

Comment:
Should this perhaps be the following?

```suggestion
lmdb-lightning-stream=yes
```

Reply:
they are equivalent, but explicit might be good